README.md

```diff
@@ -239,6 +239,58 @@ OktaAuth
   }
 ```
 
+### authoize
+
+If you already logged in to Okta and have a valid session token, you can complete authorization by calling `authoize(withSessionToken:)`. By default, this method uses the values specified in the `Okta.plist` file:
```

Comment:
Which values are we referring to here? Things like issuer and scope?

Reply:
issues, clientID, scope, redirectURI